src/bindings/pyrpr/src/pyrpr2.py

```diff
@@ -39,6 +39,25 @@ def render_update_callback(progress, data):
             self.render_update_callback = None
 
 
+class Shape(pyrpr.Shape):
+    def set_motion_transform(self, transform, transpose=True): # Blender needs matrix to be transposed
```

Comment:
I think this is ok for now. If for the future there was support for > 2 motion transforms this could be more generic like:

set_motion_transform(self, transform, time, transpose=True)

Reply:
yes

Reply:
From core team this might come sooner rather than later, not sure if you want to make this change now.

Reply:
I'd prefer to make this change, when set_motion_transform be available in RPR1. Anyway this is very easy change.